### DISCUSSION TITLE
option to allow experience stored in backpack's tank to activate mending on tools

### DISCUSSION THREAD

**Discussion by DoomSquirter** [P3pp3rF1y/SophisticatedBackpacks#297] [Feature / Idea Archive] [answered]
instead of having to make bottles of xp and throwing them at you with a dispenser, how bout some way to use the xp in the experience tank in the backpack to activate mending on tools.  either in backpack or some slots in the upgrade maybe?

**Comment by P3pp3rF1y** [comment, marked as answer]
this is implemented in the latest release - xp pump will heal the tools / armor (from xp in the tank upgrade) the same way in-world xp orbs do it in vanilla.
